app/views/RoomsListView/components/ServersList.tsx

```diff
@@ -124,24 +115,21 @@ const ServersList = () => {
 		<ServerItem
 			item={item}
 			onPress={() => select(item.id, item.version)}
-			onLongPress={() => item.id === server || remove(item.id)}
+			onDeletePress={() => remove(item.id)}
```

Comment:
`item.id === server` would prevent the current server from being removed.
I tried on your branch and it doesn't work properly, so it's probably better to keep it.